nni/experiment/config/retiarii_local.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Seems this is identical to local.pay?

Reply:
Almost same. The only difference is 
```
platform: str = 'retiarii_local'
```

This PR tries to bring least impact to existing code, so it copies the existing local training service to build retiarii_local training service.